Project-Aurora/Project-Aurora/Devices/LightFX/LightFX.cs

```diff
@@ -202,14 +205,18 @@ public override void Shutdown()
                     IsInitialized = false;
                 }
             }
+
+            return Task.CompletedTask;
```

Comment:
is this line really necessary?

Reply:
This is again avoiding compiler warn CS1998. It's also more explicit and clearer that the implementation is not actually asynchronous, IMO.